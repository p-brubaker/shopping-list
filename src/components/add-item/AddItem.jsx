import React from 'react'

export default function AddItem() {
    return <h1>Add Item</h1>
}
