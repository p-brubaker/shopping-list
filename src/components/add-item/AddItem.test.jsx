import AddItem from '../add-item/AddItem';
import { render, screen } from '@testing-library/react';

it('should render the addItem component', async () => {
  const { container } = render(<AddItem />);

  screen.getByLabelText('Item Name');
  expect(container).toMatchSnapshot();
});
