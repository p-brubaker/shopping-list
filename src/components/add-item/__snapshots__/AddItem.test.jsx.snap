// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the addItem component 1`] = `
<div>
  <form>
    <label
      for="item-name"
    >
      Item Name
    </label>
    <input
      id="item-name"
      name="item-name"
      value=""
    />
    <button
      type="submit"
    >
      Add
    </button>
  </form>
</div>
`;
